docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -59,6 +59,18 @@ Adding the following snippet to a Quadlet file extends the systemd timeout to 15
 TimeoutStartSec=900
 ```
 
+### Service Type
+
+By default, the `Type` field of the `Service` section of the Quadlet file does not need to be set.
+Quadlet will set it to `notify` for `.container` and `.kube` files and to `oneshot` for `.volume`, `.network` and `.image` files.
+
+However, the value may be explicitly set to `oneshot` for `.container` and `.kube` files when the no containers are expected
```

Comment:
```suggestion
However, `Type` may be explicitly set to `oneshot` for `.container` and `.kube` files when no containers are expected
```

Reply:
Oneshot means nothing to me in the container world. In containers it is Restart or not, but that should be handled in the systemd part not the podman part in my opinion.

Reply:
Ok now that I read up a little I know this is a systemd feature, not a kube feature. Never mind...